index.js

```diff
@@ -93,9 +94,14 @@ module.exports = (text, opts) => {
 	};
 
 	const colorizeContent = x => opts.backgroundColor ? chalk[opts.backgroundColor](x) : x;
+	const columns = termSize().columns;
 
 	text = ansiAlign(text, {align: opts.align});
 
+	if (text.length > columns) {
```

Comment:
`text.length` is not an accurate measure. Some characters can have `.length` of 3, but take up 2 columns. Better to just let `wrap-ansi` handle everything.

Reply:
@sindresorhus -Not sure what you mean. AFAIK, `wrap-ansi` does not provide an option to do a wrap at a certain column length (or to automatically wrap at terminal width), or perhaps I am just missing something here.

Reply:
https://github.com/chalk/wrap-ansi/blob/master/readme.md#wrapansiinput-columns-options

Second argument

Reply:
@SamVerschueren -Aware of that. My mistake in the previous comment. I did not explain myself well.

```
const columns = termSize().columns;
text = ansiAlign(text, {align: opts.align});

if (text.length > columns) {
  text = wrapAnsi(text, columns);
}
```

It is already being done like that. I was wondering about the `text.length` and `wrap-ansi` format;

```
text = ansiAlign(text, {align: 'center'});
text = wrapAnsi(text, columns);
```

`wrapAnsi` does not respect the `center` format, tests fail.

Reply:
Can you reverse the calls?

```js
text = wrapAnsi(text, columns);
text = ansiAlign(text, {align: 'center'});
```